visual-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "playwright",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "node ./scripts/visual-tests-build.mjs",
+    "docker:run": "docker pull mcr.microsoft.com/playwright:v1.28.1-focal && docker run -v $(INIT_CWD)/..:/tests/ -w /tests/visual-test --rm -it mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash",
+    "docker:run:win": "docker pull mcr.microsoft.com/playwright:v1.28.1-focal && docker run -v %INIT_CWD%/..:/tests/ -w /tests/visual-test --rm -it mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash",
+    "upload": "node ./scripts/visual-tests-upload.mjs",
+    "all": "node ./scripts/visual-tests-build.mjs && node ./scripts/visual-tests-upload.mjs"
```

Comment:
I'm not sure if `all` is the best name for this script.

1. It could be confused with [the `all` script from the monorepo's root.](https://github.com/handsontable/handsontable/blob/12.2.0/package.json#L16)
2. `all` doesn't mean much in this context. Maybe something more descriptive would fit this better?